pygeos/test/test_measurement.py

```diff
@@ -175,3 +175,79 @@ def test_haussdorf_distance_empty():
 def test_haussdorf_distance_densify_empty():
     actual = pygeos.hausdorff_distance(point, empty, densify=0.2)
     assert np.isnan(actual)
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 7, 0), reason="GEOS < 3.7")
+@pytest.mark.parametrize(
+    "geom1, geom2, expected",
+    [
+        # example from GEOS docs
+        (
+            pygeos.linestrings([[0, 0], [50, 200], [100, 0], [150, 200], [200, 0]]),
+            pygeos.linestrings([[0, 200], [200, 150], [0, 100], [200, 50], [0, 0]]),
+            200
+        ),
+        # another example from GEOS docs
+        (
+            pygeos.linestrings([[0, 0], [50, 200], [100, 0], [150, 200], [200, 0]]),
+            pygeos.linestrings([[0, 0], [200, 50], [0, 100], [200, 150], [0, 200]]),
+            282.842712474619
+        ),
+        # example from GEOS tests
+        (
+            pygeos.linestrings([[0, 0], [100, 0]]),
+            pygeos.linestrings([[0, 0], [50, 50], [100, 0]]),
+            70.7106781186548
+        ),
+    ],
```

Comment:
@chaitan94 Thank you for adding `parametrize`d tests.  What do you think about adding a 0-distance example, to validate that two identical geometries return 0 distance?

```
        # identical geometries should have 0 distance
        (
            pygeos.linestrings([[0, 0], [100, 0]]),
            pygeos.linestrings([[0, 0], [100, 0]]),
            0,
        ),
```

Likewise, one that demonstrates that direction really does matter:
```
        # same geometries but different curve direction results in maximum
        # distance between vertices on the lines.
        (
            pygeos.linestrings([[0, 0], [50, 200], [100, 0], [150, 200], [200, 0]]),
            pygeos.linestrings([[200, 0], [150, 200], [100, 0], [50, 200], [0, 0]]),
            200,
        ),
```

Reply:
Makes sense. I have added both of these test cases.